docs/packaging/creating-a-new-package.md

```diff
@@ -1,37 +1,101 @@
 ---
 
 # Creating a New Package
 
-This page will take you through the steps required to build a new package on Solus. For this example, we will use the command line utility [tree](https://gitlab.com/OldManProgrammer/unix-tree), which is already packaged for Solus, but we will pretend like it's not.
+This page will take you through the steps required to build a new package for the Solus package repositories. For this example, we will use the command line utility [tree](https://gitlab.com/OldManProgrammer/unix-tree), which is already packaged for Solus, but we will pretend like it's not.
 
-## Create a new Package Directory
+:::note
 
-From within your build directory, create a new directory for the new package with the package name, and create the required Makefile within that directory:
+**Please [look to see if an issue has been filed](https://github.com/getsolus/packages/issues?q=label%3A%22Package+Request%22) for the software or library you intend to package**. If there is an existing request, please add a link to it in your pull request. Ex:
+
+```
+This PR resolves software request https://github.com/getsolus/packages/issues/123
+```
+
+:::
+
+## Fork the getsolus/packages Repository / Update Your Fork
+
+### Create a Fork
+
+If you had not yet done so already, fork the repo [getsolus/packages](https://github.com/getsolus/packages) using the GitHub web UI or [`gh` cli tool](https://cli.github.com/manual/gh_repo_fork) from the `github-cli` package. It will be forked to `github.com/yourgithubaccount/packages`.
+
+### Update a Fork
+
+If you already have a fork of [getsolus/packages](https://github.com/getsolus/packages) in GitHub, log into GitHub. Make sure you're looking at the master branch. Check to see that your fork is up to date with the main repo it was forked from. If your fork indicates it is behind, use the "Sync fork" button to bring it up to date.
+
+## Clone the packages repo / update your clone
+
+### Clone the repo
+
+From within your packaging directory, clone the fork. For example `gh repo clone yourgithubaccount/packages`.
+Then, change to this directory and create a new branch.
+
+```bash
+cd /home/solus-builds/packages
+git switch -c your-branch
+```
+
+### Update an existing clone
+
+If you already have a local clone, you need to bring it up to date and create a new branch. To do so run:
+
+```bash
+cd /home/solus-builds/packages
+git switch master
+git pull
+git switch -c your-branch
+```
+
+## Create a New Package Directory
+
+From within your packaging directory, create a new directory for the new package. Use the package name as the directory name.
+
+Initialize the folder as a git repository and switch to a new branch.  This will allow you to more easily separate your work from any new changes made to the package repository, which will allow you to more easily rebase any changes if needed. Then, create the required Makefile within that directory.
 
 ```bash
 mkdir tree
 cd tree
+git init
+git switch -c my-branch
```

Comment:
These two lines are not needed. The repo is setup above.

Reply:
Just above you recommended
"Initialize a folder in the packages repository and switch to a new branch."

So it would still need to be 
```bash
mkdir tree
cd tree
git switch -c my-branch
```

(I have removed the new branch command that was before this so the user is only doing it once)